packages/server/computation_container/test/unit_test/csprng_test.cpp

```diff
@@ -159,3 +194,74 @@ TEST(CsprngTest, HowUse)
     11867235327943535202
     */
 }
+
+TEST(csprngtest, interface)
+{
+    using namespace std;
+    qmpc::random::sodium_random random;
+    auto vec1 = random(5);
+    for (auto&& a : vec1)
+    {
+        std::cout << "random vec is " << a << std::endl;
```

Comment:
Since the output alone is not automatically checked, I would like to see it changed to some kind of comparison test. Or delete it.`csprngtest/interfaceDefaultUse` and `RandomTest/newRandomTest` is too.
if the required comparison test is difficult and you do not want to delete it, I thought it would be good to move to `test/benchmark/`.